src/containers/Moods.js

```diff
@@ -26,27 +26,28 @@ const Moods = () => {
   const face = useSelector(getFace);
 
 
+  handleSelection = action => { 
```

Comment:
since you changed this from a class component to functional, this will need to be a const handleSelection instead of a method. :)